include/ClipView.h

```diff
@@ -244,7 +244,12 @@ protected slots:
 	TimePos draggedClipPos( QMouseEvent * me );
 	int knifeMarkerPos( QMouseEvent * me );
 	void setColor(const std::optional<QColor>& color);
-	//! Return true iff the clip could be split. Currently only implemented for samples
+	
+	/**
+	* Split this Clip into two clips
+	* @param pos the position of the split, relative to the start of the clip
+	* @return true iff the clip could be split
+	*/
```

Comment:
Since all clips provide splitting functionality now, this could be made pure virtual:
```c++
virtual bool splitClip(const TimePos pos) = 0;
```